lizmap/plugin.py

```diff
@@ -480,6 +480,7 @@ def write_log_message(message, tag, level):
         self.global_options['hideOverview']['widget'] = self.dlg.cbHideOverview
         self.global_options['hideNavbar']['widget'] = self.dlg.cbHideNavbar
         self.global_options['hideProject']['widget'] = self.dlg.cbHideProject
+        self.global_options['wmsSingleRequestForAllLayers']['widget'] = self.dlg.cbWmsSingleRequestForAllLayers
```

Comment:
L433, this checkbox must be added as well for LWC 3.8

Reply:
can you address me where please?

Reply:
L433 as I said, just add a new line

Reply:
done